R/shiny-recorder.R

```diff
@@ -184,6 +184,12 @@ shouldIgnoreGET <- function(path) {
   length(unlist(stringr::str_match_all(path, ignoreGET))) > 0
 }
 
+CLIENT_WS_STATE <- list(
+  UNOPENED = "UNOPENED",
+  OPEN = "OPEN",
+  CLOSED = "CLOSED"
+)
```

Comment:
Just be aware, this style of "enum" isn't often used in R. If nothing else, because `$` happily performs partial matching (e.g. `CLIENT_WS_STATE$CLOSE` would return `"CLOSED"`) which sorta defeats the purpose.

Reply:
Thanks, I think I'll keep it with that in mind, since at the very least it consolidates the set of strings.